app/views/foreman_virt_who_configure/api/v2/configs/main.json.rabl

```diff
@@ -6,4 +6,7 @@ attributes :name, :interval, :organization_id, :whitelist, :blacklist, :hypervis
            :hypervisor_type, :hypervisor_server, :hypervisor_username, :debug,
            :satellite_url, :proxy, :no_proxy, :status, :last_report_at, :out_of_date_at,
            :filter_host_parents, :exclude_host_parents
+child :http_proxy do
```

Comment:
nitpick: blocks should be surrounded by spaces (i dont think we have rubocop here to complain about it)

Reply:
Updated